client/src/lib/components/+game-ui/widgets/command-center/widget-command-center.svelte

```diff
@@ -47,33 +58,100 @@
       clearInterval(refreshInterval);
     }
   });
+
+  // Set custom title and controls if provided
+  $effect(() => {
+    if (setCustomTitle) {
+      setCustomTitle(customTitleSnippet);
+    }
+  });
 </script>
 
-<div class="command-center-widget h-full w-full flex flex-col">
-  <!-- Tab Navigation -->
-  <div class="flex gap-2 w-full justify-center mt-2 px-4">
-    <Button
-      class="w-full {activeTab === 'history' ? '' : 'opacity-50'}"
-      variant={activeTab === 'history' ? 'blue' : undefined}
+{#snippet customTitleSnippet()}
+  <div class="flex items-center gap-1">
+    <img
+      src="/ui/icons/Icon_Building1_Thin.png"
+      alt="Command Center"
+      class={cn('h-4 w-4')}
```

Comment:
cn is not required since the latest versions of svelte

```suggestion
      class="h-4 w-4"
```